CONTRIBUTING.md

```diff
@@ -0,0 +1,187 @@
+# CONTRIBUTING to Flex
+
+## Introduction
+
+Thanks for your interest in contributing to Flex! Flex's [issue tracker](https://github.com/westes/flex/issues) is on GitHub. That's the best place to find a task that needs attention. It's also the best place to check whether a problem you've found is known to the community.
+
+## Baseline Build Environment
+
+The maintenance baseline build environment is Ubuntu 20.04 LTS with at least the following packages installed:
```

Comment:
Now that INSTALL exists with proper documentation of building from scratch, should we remove this section of the contributing doc?

Reply:
Note: The INSTALL.md document didn't yet mention the flex dependency when building from git repository. Flex needs an older version of itself to bootstrap.